sdk/appconfig/azappconfig/CHANGELOG.md

```diff
@@ -1,16 +1,10 @@
 # Release History
 
-## 0.3.0 (Unreleased)
-
-### Features Added
+## 0.3.0 (2022-05-10)
```

Comment:
@jhendrixMSFT, if we release today, I'll submit this change to the PR before triggering the release pipelines. I'll keep monitoring for the `azcore` release to appear before I start. But I'll need your approval for this PR, should we release today or tomorrow.
```suggestion
## 0.3.0 (2022-05-09)
```